### PR TITLE
Adding release and cancel as an action

### LINKED ISSUE #36: Changing release date through API
Now it is only possible to change the release data manually though the website.

### PR DESCRIPTION
**Pull request by bedroesb**
- [x] Having the actions working
- [x] Adding the action updates to the updated tables
- [x] Extensive testing
- [x] Updating documentation

This PR closes #36

### CODE REVIEW

File: ena_upload/ena_upload.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from lxml import etree
 import pandas as pd
 import tempfile
-from ena_upload._version import __version__
+__version__ = 'uedje'
```
Comment (bedroesb) [inline ena_upload/ena_upload.py:21]: Reminder to change this back

**Comment by bedroesb** [comment]
When released:

```
Backend Qt5Agg is interactive backend. Turning interactive mode on.
```

Gets printed in the stdout

**Comment by bedroesb** [comment]
@bgruening I think this is ready for review :)

File: ena_upload/ena_upload.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python
 __authors__ = ["Dilmurat Yusuf", "Bert Droesbeke"]
 __copyright__ = "Copyright 2020, Dilmurat Yusuf"
+__maintainer__ = "Bert Droesbeke"
 __email__ = "Dilmurat.yusuf@gmail.com"
```
Comment (bgruening) [inline ena_upload/ena_upload.py:5]: change to your email?

**Review by bgruening — APPROVED** [review]

File: ena_upload/ena_upload.py (hunk 2 above)
Comment (bedroesb) [reply ena_upload/ena_upload.py:5]: I don't know, the biggest part of the base code is still thanks to Dilmurat, so I also don't want to steal his thunder...

Comment (bgruening) [reply ena_upload/ena_upload.py:5]: Yes, but people should probably contact you, left?

Comment (bedroesb) [reply ena_upload/ena_upload.py:5]: that is also true :) Will change it

**Pull request merged**